src/makeInfinityAdder.js

```diff
@@ -3,8 +3,24 @@
 /***
  * @return {function}
  */
-function makeInfinityAdder() {
-  // write code here
+function makeInfinityAdder(currentNum) {
+  let currentNummount = currentNum || 0;
+
+  function sum(nextNum) {
+    if (nextNum !== undefined) {
+      currentNummount += nextNum;
```

Comment:
This line uses the variable `currentNummount`, which contains a typo. It should be `currentNum` to ensure the correct variable is being updated.